### PR TITLE
Add locale package to install image

### PR DESCRIPTION
**Pull request by YuryLysov**
## Description
Added locale package to install image

## Why do we need it, and what problem does it solve?
Resolve issue #10713

## What is the expected result?
Using install image, users won't see a warning about locale

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ci
type: fix
summary: Add locale package to install image, to fix warning about locale.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @YuryLysov. I've started the workflow for label 'e2e/run/aws'...

File: .werf/werf-dev-install.yaml [hunk 1]
```diff
@@ -103,3 +103,5 @@ shell:
 
     echo 'eval "$(dhctl --completion-script-bash)"' >> /etc/bashrc
     echo '{{- env "CI_COMMIT_TAG" | default "dev" }}' > /deckhouse/version
+
+    apt-get update && apt-get install -y locale && apt-get clean
```
Comment (sprait) [inline .werf/werf-dev-install.yaml:107]: move this commands in `beforeInstall` stage pls

**Review by sprait — COMMENTED** [review]
one comment

File: .werf/werf-dev-install.yaml (hunk 1 above)
Comment (YuryLysov) [reply .werf/werf-dev-install.yaml]: Done

**Review by sprait — APPROVED** [review]

**Comment by unreturned** [comment]
@YuryLysov please, add also something like this `find /var/lib/apt/ /var/cache/apt/ -type f -delete`, because `apt-get clean` its not enough to remove all downloaded files (like lists):

```
[deckhouse] root@a476fc1df4c6 / # du -hs /var/lib/apt/ /var/cache/apt/
258M	/var/lib/apt/
181M	/var/cache/apt/
[deckhouse] root@a476fc1df4c6 / # find /var/lib/apt/ /var/cache/apt/
/var/lib/apt/
/var/lib/apt/prefetch
/var/lib/apt/lists
/var/lib/apt/lists/partial
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_noarch_base_release
/var/lib/apt/lists/lock
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64_base_pkglist.classic
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64-i586_base_release
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64_base_release
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64_base_release.classic
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64-i586_base_pkglist.classic
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_x86%5f64-i586_base_release.classic
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_noarch_base_pkglist.classic
/var/lib/apt/lists/ftp.altlinux.org_pub_distributions_ALTLinux_p10_branch_noarch_base_release.classic
/var/cache/apt/
/var/cache/apt/archives
/var/cache/apt/archives/partial
/var/cache/apt/archives/lock
/var/cache/apt/pkgcache.bin
/var/cache/apt/srcpkgcache.bin
```

And this not necessary files dramatically increases image size:
```
docker images -a | grep install
dev-registry.deckhouse.io/sys/deckhouse-oss/install   main          dbe354c94542   4 hours ago    1.18GB
dev-registry.deckhouse.io/sys/deckhouse-oss/install   pr10803       1d9dc0106165   12 hours ago   1.89GB
```

**Review by name212 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/openstack'...

**Pull request merged**